src/inputs/TextFieldBase.tsx

```diff
@@ -120,6 +120,7 @@ export function TextFieldBase<X extends Only<TextFieldXss, X>>(props: TextFieldB
         .if(compact)
         .mhPx(compactFieldHeight - maybeSmaller).$,
       ...Css.gray900.if(contrast).white.$,
+      ...(borderless ? Css.px1.$ : {}),
```

Comment:
Maybe inline comment `// Make read-only fields vertically line up with editable fields in tables`?